app/models/direct_file_data.rb

```diff
@@ -764,6 +769,7 @@ def attributes
       mailing_apartment
       mailing_zip
       cell_phone_number
+      phone_number
```

Comment:
having to add this really got me in terms of adding the ability to edit via the fed info page. could we maybe generate this from `SELECTORS` instead?

Reply:
that's a great idea